x/kavadist/simulation/genesis.go

```diff
@@ -0,0 +1,84 @@
+package simulation
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	"github.com/cosmos/cosmos-sdk/x/simulation"
+
+	"github.com/kava-labs/kava/x/kavadist/types"
+)
+
+// SecondsPerYear is the number of seconds in a year
+const SecondsPerYear = 31536000
+
+// BaseAprPadding prevents the calculated SPR inflation rate from being 0.0
+const BaseAprPadding = "0.000000000100000000"
```

Comment:
To set minimum inflation to 10%, the BaseAprPadding would be `0.000000003022265980`

Reply:
https://github.com/Kava-Labs/kava/pull/439/commits/1b5f2889b5741ed550ff2f45a71f5a0e2d090b4e